packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -225,6 +226,18 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
   AppLifecycleState get lifecycleState => _lifecycleState;
   AppLifecycleState _lifecycleState;
 
+  /// Initializes the [lifecycleState] with the data from the window.
+  ///
+  /// An optional [testWindow] may be provided to supply test values.
+  ///
+  /// Once the lifecycleState is initialized, it cannot be initialized again.
```

Comment:
If the name of the method changes, then probably should change "initialized" here, too.  Also, it's worth pointing out why this can only happen once.  Since the lifecycle changes over time, it may surprise the reader that this cannot be run again to get an update.